src/org/daisy/dotify/api/formatter/TableOfContents.java

```diff
@@ -21,4 +21,23 @@
 	 */
 	public void endEntry();
 	
+    /**
+	 * Starts a new entry-on-resumed with the supplied properties.
+	 * 
+	 * @param range the range of this entry on resumed
+	 */
+	public void startEntryOnResumed(String range);
+	
+	/**
+	 * Ends the current entry-on-resumed.
+	 */
+	public void endEntryOnResumed();
+	
+    /**
+     * Get the entry-on-resumed.
+     * 
+     * @return the entry-on-resumed.
+     */
+    public FormatterCore getEntryOnResumed();
```

Comment:
Not sure I understand what this is for?

Reply:
After having reviewed the dotify.formatter.impl PR I understand why you did it. But this really doesn't belong in the API. In fact I don't think it is needed at all. See my comment in the other PR.

Reply:
I have removed this interface method because it was not needed anymore in the simplified solution.